### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Sniy** [ldeb/dayzstats#1]
Hi ,

It works without Discord ? Thanks

**Comment by ldeblasis** [comment]
Hello, yes it does, Discord system is not yet plugged. It will be optionnal.
But be aware, it's still in progress and not yet working with a database, so, at the moment, it will only parse 1 log file

**Comment by ldeb** [comment]
Hello, yes it does, Discord system is not yet plugged. It will be optionnal.
But be aware, it's still work in progress and not yet working with a database, so, at the moment, it will only parse the current log file.

**Comment by Sniy** [comment]
Hi . Okay many thanks .
Good job .

**Comment by iamsquaredude** [comment]
Great job and easy to get up and running. Changed the map to a more recent version and it looks great https://the-unclan.com/KillsMarch/

**Comment by ldeb** [comment]
Nice to see it already in action! 
The sortable table was broken, Check out my last commit!

Also, beware, your new map size is around 25Mo! I prepared a lighter .jpg version (2.7Mo).
I am curious, have you verified the precision of the dots position on it ? 
If you made changes to the [`coord2px()`](https://github.com/ldeb/dayzstats/blob/master/inc/functions.php#L175) function, I'd be glad to know which ones, thank you.

**Comment by iamsquaredude** [comment]
Yeah I was just using the map pulled from the game to see if I could get everything working, I'll be updating it to your newest version. I did make changes to the `coord2px() `  and it seems to line up with kills I've plotted on izurvive
```
function coord2px($worldspace){
  $y_diff = 15360;
  $coords = explode(', ', $worldspace);
  if( count($coords)==3 ) {
    $result = array();
    $result[0] = floatval( $coords[0] ) / 3.743;
    $result[1] = ($y_diff - floatval( $coords[1] )) / 3.743;
    $result[2] = floatval( $coords[2] );
    return $result;
  } else {
    return array(0,0,0);
  }
}
```
